### PR TITLE
Staging to main to merge diversity metrics and update the folder names

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

**Comment by YanZhangADS** [comment]
The "Novelty" section of als_movielens_diversity_metrics.ipynb notebook contain unknown characters in the formula. @anargyri

**Comment by miguelgfierro** [comment]
> The "Novelty" section of als_movielens_diversity_metrics.ipynb notebook contain unknown characters in the formula

Andreas is out, is there somebody that could take that task?

**Comment by YanZhangADS** [comment]
> > The "Novelty" section of als_movielens_diversity_metrics.ipynb notebook contain unknown characters in the formula
> 
> Andreas is out, is there somebody that could take that task?

I have made the change and submitted the PR for review https://github.com/microsoft/recommenders/pull/1488

File: tests/integration/remmenders/dataset/test_criteo.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import os
```
Comment (laserprec) [inline tests/integration/remmenders/dataset/test_criteo.py:4]: @miguelgfierro I think the test folder `integraton/remmenders` is misspelled 😄

Comment (miguelgfierro) [reply tests/integration/remmenders/dataset/test_criteo.py]: good catch, thanks!
It's fixed now

**Comment by wutaomsft** [comment]
curious why we are still using master branch?

**Comment by loomlike** [comment]
> curious why we are still using master branch?

Actually the merging is into `main`. Only the title says `master` (typo). FYI I checked `master` branch hasn't been used after we deprecated it so I just removed it @miguelgfierro.

**Comment by loomlike** [comment]
Accidentally closed the PR. reopening it lol

**Review by loomlike — APPROVED** [review]
lgtm

**Comment by anargyri** [comment]
Since this merge includes the renaming of the folders, I suggest we add a note about this on the front page. Also, how about tagging this as a release?

**Comment by miguelgfierro** [comment]
> Also, how about tagging this as a release?

agree

@laserprec with the changes you are making to the build pipeline, is it ok to merge this PR?

**Comment by anargyri** [comment]
> > Also, how about tagging this as a release?
> 
> agree
> 
> @laserprec with the changes you are making to the build pipeline, is it ok to merge this PR?

I have prepared PR https://github.com/microsoft/recommenders/pull/1500 about this, you could edit as you like and merge into staging.

**Comment by laserprec** [comment]
@miguelgfierro Yup! These build changes are standalone, and there weren't any source code changes introduced. One thing it may change is that PR-ing into `main` and `staging` will trigger this new PR-Gate in GitHub Actions to run in addition to our existing CIs in ADO. 

All of the GitHub Actions are non-blocking until we this new CI infra achieve feature-parity with out existing ADO CIs.

**Comment by miguelgfierro** [comment]
I'll merge this then

**Pull request merged**